CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.1 (Apr 7, 2022)
+
+IMPROVEMENTS:
+
+* Added VCS remote repository resource - `artifactory_remote_vcs_repository` [GH-394]
```

Comment:
Please include the version of rt this was tested against

Reply:
BTW: the GH ticket should be linked. Can you figure that out?

Reply:
Usually, we link only the PR number in CHANGELOG.md

Reply:
GitHub will auto generate the link to the PR because we are using the format `[GH-394]` in the text.